README.md

```diff
@@ -220,6 +220,7 @@ Prerequisites for building Mission Control:
 On Linux you can use the build.sh script to build JMC:
 ```
 usage: call ./build.sh with the following options:
+   --installCore to install the core bits
```

Comment:
How about "to install the core artifacts"?

Reply:
Thanks for reviewing, should be done now.